File: VB/MyTabControl/MyXtraTabPageCollection.vb
Imports Microsoft.VisualBasic
Imports DevExpress.XtraTab
Imports System
Imports System.Collections.Generic
Imports System.Linq
Imports System.Text

Namespace Example_HidePageHeader.MyTabControl
	Public Class MyXtraTabPageCollection
		Inherits XtraTabPageCollection
		Public Sub New(ByVal tabControl As XtraTabControl)
			MyBase.New(tabControl)
		End Sub
		Protected Overrides Function CreatePage() As XtraTabPage
			Return New MyXtraTabPage()
		End Function
	End Class
End Namespace

File: CS/MyTabControl/MyXtraTabPageCollection.cs
using DevExpress.XtraTab;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace Example_HidePageHeader.MyTabControl {
    public class MyXtraTabPageCollection : XtraTabPageCollection {
        public MyXtraTabPageCollection(XtraTabControl tabControl)
            : base(tabControl) {
        }
        protected override XtraTabPage CreatePage() {
            return new MyXtraTabPage();
        }
    }
}
